bigchaindb_driver/driver.py

```diff
@@ -31,10 +29,18 @@ def __init__(self,
                 ``'http://localhost:9984/api/v1'``.
             transport_class: Optional transport class to use.
                 Defaults to :class:`~bigchaindb_driver.transport.Transport`.
+            headers (dict): Optional headers that will be passed to the
+                ``transport_class`` instantiation, which will in turn
+                pass it to the :class:`~.connection.Connection`
+                instances. Please note that the given headers will be
+                sent on each request. To pass headers only on a
```

Comment:
I wouldn't bother with the details about how it's passed, just that headers sent this way will be sent with each request.

```python
headers (dict, optional): Headers to be pass with each request from the driver. If you'd like to pass headers only on a per-request basis...